common/djangoapps/third_party_auth/models.py

```diff
@@ -243,15 +244,7 @@ def enabled_for_current_site(self):
         """
         Determines if the provider is able to be used with the current site.
         """
-        if not self.enabled:
-            return False
-        if configuration_helpers.is_site_configuration_enabled():
-            # If there's a site enabled for this request, then return True if the site matches.
-            config = configuration_helpers.get_current_site_configuration()
-            return config.site == self.site
-        # If not viewing a site, return True if the site is unset (which signifies that the provider is intended for
```

Comment:
This code is no longer here. It's much simpler now that the sites aren't nullable. :)